SECURITY.md

```diff
@@ -3,6 +3,23 @@
 
 Join the [kubestellar-security-announce](https://groups.google.com/u/1/g/kubestellar-security-announce) group for emails about security and major API announcements.
 
+## Dependencies Policy
+
+KubeStellar manages its dependencies with the following policy:
+
+- **Dependency Detection:** We use [Dependabot](https://github.com/dependabot) to automatically check for and propose updates to dependencies in Go modules, Python requirements, Dockerfiles, and Helm charts. Dependabot PRs serve as prompts but are not automatically accepted.
+- **Update Process:** After Dependabot creates a PR, maintainers wait for potential issues to surface before proceeding. They then create their own PR that follows our [GitHub Action reference discipline](https://github.com/kubestellar/kubestellar/blob/main/CONTRIBUTING.md#github-action-reference-discipline) and other established practices.
+- **Review Process:** All dependency update pull requests are subject to the same [review process](https://github.com/kubestellar/kubestellar/blob/main/CONTRIBUTING.md#pull-requests) as other code changes. Maintainers verify that updates do not introduce breaking changes or known vulnerabilities before merging.
+- **Vulnerability Checking:** Before merging dependency updates, maintainers perform comprehensive security assessments:
+  - **Security Scanning:** Use available security scanning tools to check for known vulnerabilities in new dependency versions
```

Comment:
What tools are those?

Note that KubeStellar imports several kinds of things. 
- Go-language packages
- Pre-built binaries
- Container images
- Helm charts
- GitHub Actions